src/forms/TopComponent/Views/ResponsiveView.tsx

```diff
@@ -17,7 +17,7 @@ import TitleWrapper from "../Utils/TitleWrapper";
 import { BaseTopComponentProps, TopComponentProps } from "../TopComponentTypes";
 
 const ResponsiveViewColumn = styled(StyledColumn)`
-	padding: ${pr => pr.sections ? "24px 20px 0px 20px" : "24px 20px 20px 20px"};
+	padding: ${pr => pr.sections ? "24px 24px 0px 24px" : "24px 24px 16px 24px"};

```

Comment:
Now that almost all variants will share the same sizes, spaces, and styling, I'm confident we could move all of this code into "StyledColumn" and delete "ResponsiveViewColumn"